Dockerfile

```diff
@@ -14,7 +14,6 @@
 
 FROM golang:1.14-alpine AS build
 WORKDIR /work
-RUN apk --no-cache add build-base git gcc
 
 ENV USER=secuser
```

Comment:
Conversely, we could remove the entire first stage image.

Reply:
See https://github.com/saschagrunert/seccomp-operator/pull/28#issuecomment-651650217